R/GeoLift.R

```diff
@@ -741,9 +741,10 @@ GeoLiftPower <- function(data,
   max_time <- max(data$time)
   data$location <- tolower(data$location)
 
-  if (max(treatment_periods)/max_time > 0.8){
-    message(paste0("Warning: Small pre-treatment period.
-                   \nTthe treatment is larger that 80% of all available data."))
+  # Small Pre-treatment Periods
+  if (max_time/max(treatment_periods) < 4){
+    message(paste0("Warning: Small pre-treatment period!.
+                   \nIt's recommended to have at least 4x pre-treatment periods for each treatment period.\n"))
```

Comment:
Artur, a couple of quick points.
- If we are starting the message with "Warning" isn't it better for the comment to be a warning instead of a message?
- if we are saying that we are looking at the pre-treatment period, shouldn't we adjust the inequation slightly for it to actually be looking only at the pre-treatment period? Cause right now its looking at the entire period.  I was thinking of something like: `if ( (max_time - max(treatment_periods)) / max(treatment_periods) < 4)...`.  

What do you think?

Reply:
Same comment for all times where the filter changed.

Reply:
Thanks for the feedback Nico:

- Changing the `message` to a `warning` is a good idea. The only reason I didn't do it is to prevent it from appearing as the `last.warning` in case something else fails in the code and the user is trying to troubleshoot with `warnings()`. My thinking was that while short pre-treatment periods are not ideal, they're theoretically possible for short tests?
- I think we're doing pretty much the same thing with both the current and your suggested inequality (your's is just subtracting 1 from the ratio of T/t.

Reply:
- Ok, Caution: small sounds good! 
- Sure, but then should we change the wording? Its not the pre-treatment period if we don't substract 1 right? Sorry for being super picky.